test/clone.test.js

```diff
@@ -0,0 +1,18 @@
+const assert = require('assert');
+const math = require('mathjs');
+
+const clone = require('../lib/util/clone');
+
+function testCloner(expNode, new_node) {
+  it ('creates a clone of a node', function () {
+    assert.equal(clone(math.parse(expNode)), new_node);
```

Comment:
wait, does this pass?

`new_node` would be a string or integer, and `clone(math.parse(expNode))` would be a node